example.cpp

```diff
@@ -1,64 +1,99 @@
+// ModbusPP.cpp : This file contains the 'main' function. Program execution begins and ends there.
+//
+
 //
 // Created by Fanzhe on 5/29/2017.
 //
 
-#include "modbus.h"
+#include "../modbuspp-master/modbus.h"
+#include "../modbuspp-master/modbus_exception.h"
```

Comment:
This seems like that should not have ended up in this commit. Those inclusion paths will not work for other people.

For the rest I'd be looking forward for this to get merged as I am planning to use this library on Windows soon.

Reply:
Yes In general the example file should be independent from the underlying definition.